docs/reference-versions.html

```diff
@@ -19,4 +19,4 @@ <h2>Available API References</h2>
 	<li><a href='reference-1.6.0.html'>API reference for 1.6.0</a>
 	<li><a href='reference-1.7.1.html'>API reference for 1.7.1</a>
 	<li><a href='reference.html'>API reference for 1.8.0</a>
-</ul></p>
+</ul>

```

Comment:
Off-topic change to fix an HTML error.

`<p><ul>...</ul></p>` is rendered as

```html
<p></p>
<ul>...</ul>
<p></p>
```

in the browser. This change doesn't affect the page visually.